utility/scriptConfig.py

```diff
@@ -103,3 +118,6 @@ def get_args_list(self) -> t.Dict:
 
     def get_help(self) -> Help:
         return self.help
+
+    def copy(self):
+        return ScriptConfig(**self.copy_info)
```

Comment:
No, it wouldn't work, because when ScriptConfig is initialized, it changes the initialization data that is passed to it. The example is args.pop('flag') in __add_args. So, if we use the data, that was already processed by first ScriptConfig initialization, then we will have errors on second ScriptConfig initialization (no 'flag' in args already)

Reply:
Ah ok, I see.